### PR TITLE
(docs) Update README

### PR DESCRIPTION
**Pull request by denniskigen**
# Requirements

- [x] This PR has a title that briefly describes the work done, including the ticket number if there is a ticket.
- [ ] My work conforms to the [OpenMRS 3.0 Styleguide](https://om.rs/styleguide) and [design documentation](https://zeroheight.com/23a080e38/p/880723-introduction).
- [ ] My work includes tests or is validated by existing tests.

## Summary

Updates the README file for this repo with the intention of providing more up to date content and cleaning up cruft.

## Screenshots

*None*

## Related Issue

*None*

## Other

*None*

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,429 +1,139 @@
...
-
-`$ yarn`
-
-`$ yarn start`
-
-### Build the library by running:
-
-`$ yarn run build:lib`
-
-### Linking ngx-formentry with form-entry-app in patient chart
```
Comment (CynthiaKamau) [inline README.md:95]: I think this part was important

File: README.md [hunk 2]
```diff
@@ -1,429 +1,139 @@
...
+# cd into the dist directory
+cd dist/ngx-formentry
 
-`$ git commit -m 'Bump <Version>'`
+# Create an empty lockfile (yarn 3+ requires this)
+touch yarn.lock
 
-`$ git tag <Version>`
+# Copy the absolute path to the built library
+pwd | pbcopy
```
Comment (ibacher) [inline README.md:120]: `pbcopy` and `pbpaste` are macOS only. The Linux equivalents are something like:

`pbcopy`
```
xclip -selection clipboard
```

And:

`pbpaste`
```
xclip -out
```

File: README.md [hunk 3]
```diff
@@ -1,429 +1,139 @@
...
 
-Reset branch so you don't commit the dist to the src repository
+# Using the O3 Patient Chart's Form Entry app as an example
+cd ~/openmrs-esm-patient-chart/packages/esm-form-entry-app
 
-```
+# Link the library
+yarn add `pbpaste`
```
Comment (ibacher) [inline README.md:126]: Shouldn't this be `yarn link`?

File: README.md (hunk 1 above)
Comment (ibacher) [reply README.md:95]: It's still there, just moved around.

File: .vscode/settings.json [hunk 4]
```diff
@@ -0,0 +1,3 @@
+{
+  "eslint.workingDirectories": ["src"]
+}
```
Comment (ibacher) [inline .vscode/settings.json:3]: I'm always a little hesitant about committing editor-specific configurations. By itself, this is actually worthwhile, but I worry about what other cruft might get added to this file.

**Review by ibacher — COMMENTED** [review]
Thank you! This is very helpful at cleaning things up! Couple of minor comments.

File: README.md (hunk 3 above)
Comment (denniskigen) [reply README.md:126]: Either works, apparently.

File: README.md (hunk 2 above)
Comment (denniskigen) [reply README.md:120]: So, would that be: `xclip -selection clipboard && xclip -out`?

Comment (ibacher) [reply README.md:120]: Well, `pwd | xclip -selection clipboard` and then ``yarn add `xclip -out` ``

**Review by ibacher — APPROVED** [review]

**Pull request merged**